src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -165,7 +165,7 @@ class TransactionDetailsComponent extends React.Component<Props, State> {
 
     Airship.show(bridge => (
       <AdvancedDetailsModal bridge={bridge} transaction={edgeTransaction} url={sprintf(wallet.currencyInfo.transactionExplorer, edgeTransaction.txid)} />
-    ))
+    )).catch(err => showError(err))
```

Comment:
Drop the `catch`, and just `await` the `Airship.show`

Reply:
`openAdvancedDetails` is used by a non-async callback event. I removed the async keyword instead.